x-pack/plugins/monitoring/server/lib/apm/get_apms.ts

```diff
@@ -90,7 +90,7 @@ export function handleResponse(response: ElasticsearchResponse, start: number, e
       memory:
         hit._source.beats_stats?.metrics?.beat?.memstats?.memory_alloc ??
         hit._source.beat?.stats?.memstats?.memory?.alloc,
-      cgroup_memory: hit._source.beats_stats?.metrics?.beat?.cgroup?.memory.mem.usage.bytes,
+      cgroup_memory: hit._source.beats_stats?.metrics?.beat?.cgroup?.mem.usage.bytes,
```

Comment:
As an example, here you'd need to check if `hit` is a Internal Collection document (use `hit._source.beats_stats.metrics.beat?.cgroup.memory.mem.usage.bytes`) or a Metricbeat Collection document (use `hit._source.beat.stats.cgroup.memory.mem.usage.bytes`).

We tend to just use `??` to fallback to the right value, like [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/monitoring/server/lib/cluster/get_cluster_status.ts#L17-L21).